ordered logistic regression.ipynb

```diff
@@ -16,34 +16,50 @@
    "outputs": [],
```

Comment:
<p>I don't think we need the try/except here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MarcoGorelli/pydataglobal-21/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
This is just for the case of someone trying (unsuccessfully) to install the Jax package (e.g., Windows user).
I am one of those cases, and Jax seemed nearly impossible to compile on Windows. Therefore, I had to operate everything from a virtual Ubuntu OS (by means of the Windows WSL), just to test this Jupyter.

Reply:
numpyro won't work without jax though, so this is fine

Reply:
I see. I will update the PR by dropping this Try/Except clause, then.